.github/workflows/offices-cleanup-code-resharper.yml

```diff
@@ -0,0 +1,53 @@
+name: ReSharper CLI CleanupCode Offices
+
+on: [ push ]
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    name: Cleanup Code Offices
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 8.0.x
+
+      - name: Restore Dependencies
+        run: dotnet restore Offices/Offices.sln
+          
+      - name: Cleanup Code
+        id: cleanup
+        uses: ArturWincenciak/ReSharper_CleanupCode@v3.0
+        with:
+          solution: 'Offices/Offices.sln'
+          fail_on_reformat_needed: 'yes'
+          jb_cleanup_code_arg: '--verbosity=INFO'
+
+  inspection:
+    runs-on: ubuntu-latest
+    name: Inspect Code Offices
+    needs: cleanup
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 8.x
+
+      - name: Restore Dependencies
+        run: dotnet restore Offices/Offices.sln
+
+      - name: Inspect code
```

Comment:
Is it possible to combine 2 steps (cleanup and inspection) to avoid doing the same work twice?

Reply:
Fixed